WhaleSpotting/Models/Request/LikeRequest.cs

```diff
@@ -0,0 +1,9 @@
+namespace WhaleSpotting.Models.Request;
+
+public class LikeRequest
+{
+    public int WhaleSightingId { get; set; }
+    public int UserId { get; set; }
```

Comment:
We will actually only need to get the Sighting ID from the request as we will get the user from the AuthHeader